src/main/scala/com/linagora/openpaas/gatling/unifiedinbox/scenari/InboxScenari.scala

```diff
@@ -77,7 +77,7 @@ object InboxScenari {
     exec(getMailboxes)
       .exec(getMessageList)
       .repeat(3)(exec(AvatarsSteps.search(UsernameSessionParam, withRandomDisplayName = true)))
-      .pause(60.seconds))
+      .pause(40.seconds, 60.seconds))
```

Comment:
why? this is exactly occuring every 60 seconds.

If you don't want all users to do this at the same time, please add one jitter at the start of the scenario

Reply:
(or better a rampUp?)

Reply:
Ok, i tought a jitter would be good here. But you are right.
No need to add one at the start of the scenario, as the simulation ramp the users during a duration.
We can close this PR.